.well-known/pgpKey.txt

```diff
@@ -0,0 +1,58 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----

```

Comment:
The pubkey exposes your email btw, since it's directly embedded. That's why I suggested generating a fresh one for the group

Reply:
Wouldn't a group key do the same, unless we created a throwaway email account?

Reply:
That's what I was suggesting, yes

Reply:
Not too worried about email leakage (as you will have to verify signature in order to get to it), and I don't see email scraping bots going that far... Also, already been added to git (so will be able to see it in previous git commits). Can implement group account if needed, but not too bothered - up to you?

Reply:
If you're not fussed then it should be fine :+1: